interp/constrexpr_ops.mli

```diff
@@ -58,6 +58,11 @@ val mkCLambdaN : Loc.t -> local_binder list -> constr_expr -> constr_expr
 val mkCProdN : Loc.t -> local_binder list -> constr_expr -> constr_expr
 (** Same as [prod_constr_expr], with location *)
 
+val fresh_var_fwd : (Names.Id.t list -> Constrexpr.constr_expr -> Names.Id.t) ref
```

Comment:
I don't think it is that much of a problem. There are plenty of those in Coq. That's why `Hook` was devised.